packages/namegraph-sdk/README.md

```diff
@@ -0,0 +1,138 @@
+# NameGraph SDK
+
+A TypeScript SDK for interacting with the NameGenerator APIs, providing access to the world's largest collection of word-wiki names. This SDK enables easy integration with the NameGenerator API endpoints for name and collection suggestions.
+
+## Installation
+
+```bash
+npm install namegraph-sdk
+# or
+yarn add namegraph-sdk
+```
+
+## Quick Start
+
+```typescript
+import { createClient } from "namegraph-sdk";
+
+// Create a client instance
+const client = createClient();
```

Comment:
```suggestion
const client = createNameGraphClient();
```

This is verbose but I think it's important to make it more explicit since there's so many different interpretations of createClient.

Suggesting to rename this everywhere.

Reply:
@FrancoAguzzi Hey this is still an open action item. Thanks